params/config_classic.go

```diff
@@ -81,8 +81,9 @@ var (
 		EIP2028FBlock: big.NewInt(10_500_839),
 		EIP2200FBlock: big.NewInt(10_500_839), // RePetersburg (=~ re-1283)
 
-		ECBP1100FBlock: big.NewInt(11_380_000), // ETA 09 Oct 2020
-		ECIP1099FBlock: big.NewInt(11_700_000), // Etchash (DAG size limit)
+		ECBP1100FBlock:           big.NewInt(11_380_000), // ETA 09 Oct 2020
+		ECBP1100DeactivateFBlock: big.NewInt(69_420_123),
```

Comment:
Let's not forget to set the block_number here or we wait for this PR to be merged and set it at #595

Reply:
I just set it here https://github.com/etclabscore/core-geth/pull/592/commits/907295dcdbcc66bf4474eae22f3341c0941ea3b9